database/seeders/AdminsTableSeeder.php

```diff
@@ -18,7 +18,7 @@ public function run()
             //Name, email, password, type
             'name'     => 'Alex Legard',
             'email'    => 'alexlegard3@gmail.com',
-            'password' => '$2y$10$LwVswuTOjNfd2YjSDnIy1OqbI2cRJSZMwZP8mgS.GP65CoXyU3KR2',
+            'password' => \Hash::make('password'),
```

Comment:
Doing this will allow others downloading the code to login.

Reply:
Another question I had about this though is why are there two account systems? One for admins and one for users? I feel like they can be merged together.

Reply:
Yes, I want people downloading the code to login.

When I was creating this website for my internship, I was asked to make a separate login guard for admins, and also a third one for super admins. So that is the way I did it.

Reply:
And by the way, the reason the passwords look like real passwords instead of just 'password' is to create a sense of realism. It feels less authentic when you just type in "password".